msd_module/msd_policy/main.tf

```diff
@@ -0,0 +1,33 @@
+module "inbound" {
```

Comment:
if it is user facing module - rename it to `msd_inbound_policy` the same for outbound